Makefile

```diff
@@ -5,22 +5,35 @@ CHANNEL_TO_TEST?=test
 VERSION?=
 SHELLCHECK_EXCLUSIONS=$(addprefix -e, SC1091 SC1117)
 SHELLCHECK=docker run --rm -v "$(CURDIR)":/v -w /v koalaman/shellcheck $(SHELLCHECK_EXCLUSIONS)
+CLEAN := *~
+
+.PHONY: deps
+deps: dirs docker.sh
+
+CLEAN += build
+.PHONY: dirs
+dirs:
+	mkdir -p build
+
+CLEAN += docker.sh
+docker.sh:
```

Comment:
This should actually just be `build/docker.sh`

And then the rule could be defined as:

```make
.PHONY: build
build: build/docker.sh

build/docker.sh:
	mkdir -p build
	@HTML='$(shell cat index.html)' envsubst < install.sh >> docker.sh
```

Seems a bit convoluted to do it the way you're doing it here.